-1

Tested at: 11a5395c3916d8614e045568cf18d0e2efa48edd


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/f56dad19aae2b1e9c39069555e360e4ea6730bcc
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19786/21555/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19786/21555/git-merge-result

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19786/21555/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_3_X_2017-07-17-2300/src/L1Trigger/L1TMuonEndCap/src/PtAssignmentUnit.cc 
>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_3_X_2017-07-17-2300/src/L1Trigger/L1TMuonEndCap/src/PtRefinementUnitFactory.cc 
>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_3_X_2017-07-17-2300/src/L1Trigger/L1TMuonEndCap/src/ForestHelper.cc 
>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_3_X_2017-07-17-2300/src/L1Trigger/L1TMuonEndCap/src/EndCapParamsHelper.cc 
>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_3_X_2017-07-17-2300/src/L1Trigger/L1TMuonEndCap/src/PatternRecognition.cc 
error: class 'csctf::TrackStub' has a different checksum for ClassVersion 11. Increment ClassVersion to 12 and assign it to checksum 1100843423
Suggestion: You can run 'scram build updateclassversion' to generate src/DataFormats/L1CSCTrackFinder/src/classes_def.xml.generated with updated ClassVersion
gmake: *** [tmp/slc6_amd64_gcc630/src/DataFormats/L1CSCTrackFinder/src/DataFormatsL1CSCTrackFinder/libDataFormatsL1CSCTrackFinder.so] Error 1
Leaving library rule at DataFormats/L1CSCTrackFinder
>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_3_X_2017-07-17-2300/src/L1Trigger/L1TMuonEndCap/src/Tree.cc 
>> Compiling  /build/cmsbld/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_3_X_2017-07-17-2300/src/L1Trigger/L1TMuonEndCap/src/Utilities.cc 
</pre>


The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/f56dad19aae2b1e9c39069555e360e4ea6730bcc
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19786/21555/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-19786/21555/git-merge-result

--- a/DataFormats/L1CSCTrackFinder/src/classes_def.xml
+++ b/DataFormats/L1CSCTrackFinder/src/classes_def.xml
@@ -13,7 +13,8 @@
     <version ClassVersion="10" checksum="3445513287"/>
    </class>
 
-   <class name="csctf::TrackStub" ClassVersion="11">
+   <class name="csctf::TrackStub" ClassVersion="12">
+    <version ClassVersion="12" checksum="1100843423"/>
     <version ClassVersion="11" checksum="2137081572"/>
     <version ClassVersion="10" checksum="2767475416"/>
      <field name="thePhiBinning" transient="true"/>

